lib/api/multiObjectDelete.js

```diff
@@ -269,6 +269,8 @@ export function getObjMetadataAndDelete(authInfo, canonicalID, request,
             // return DeleteMarkerVersionId equals the versionID of the marker
             // you just generated and DeleteMarker tag equals true
             if (deleteInfo.newDeleteMarker) {
+                // Adding a delete marker should add one to numberOfObjects
```

Comment:
Thanks for the feedback. Yes, with the help of @electrachong, I found that we could simply decrement `numOfObjects` for each delete marker that is put. This handles a couple cases:
* If only two delete markers are put, this value becomes `-2`, and the utapi metrics for `numberOfObjects` is incremented by two.
* If versions are deleted and delete markers are put in the same `multiObjectDelete` operation, then we end up with the correct number of objects to either increment or decrement for Utapi.

I wonder if this choice goes against the indended use of Redis commands (i.e., using the `decrby` command with a negative number to increment a value. What do you think @rahulreddy? Perhaps we could add a case in Utapi to check for the value being > 0 or < 0 and increment and decrement accordingly, for clarity's sake.